### DISCUSSION TITLE
Is there any hard dependency on visual studio 17.9?

### DISCUSSION THREAD

**Discussion by ElectricVampire** [Giorgi/EFCore.Visualizer#33] [Q&A] [answered]
Is there any hard dependency on visual studio 17.9? Can it work with lower versions of visual studio?
Many developers I know and work with are stuck with lower version of VS2022, and it's difficult to procure higher version in Org.
Is it possible if it can work with 17.8.

Thank you for awesome extension. Also want to share that I have started some work on enhancing it for Oracle, Will create a PR soon

**Comment by Giorgi** [comment, marked as answer]
Yes, there is a hard dependency on 17.9

**Reply by ElectricVampire** [reply]
Ok, But I not able to get extension working with VS 2022 17.11, I thought it should work on any version higher than 17.8
![image](https://github.com/user-attachments/assets/b163fcf8-a682-4cda-9bcf-27fc50ea987a)
![image](https://github.com/user-attachments/assets/3d995359-9589-4990-9420-1189589eb654)

**Reply by Giorgi** [reply]
Can you add a where clause to it and see if that helps? It works with VS 17.9.0 or newer

**Reply by ElectricVampire** [reply]
Worked

**Reply by Giorgi** [reply]
Can you share the project for which it didn't work?

**Reply by ElectricVampire** [reply]
It doesn't work when one do ".ToList" directly on the DbSet, when I debugged I found that, it could be because we never get a hook for IQueryable , it works when I put something in when gives me IQueryable
I can share the project as well, if you still want

**Reply by Giorgi** [reply]
Yes, please share the project

**Reply by ElectricVampire** [reply]
Please check the project here : https://github.com/ElectricVampire/EfCoreTesting

You will be able to reproduce like following:
![image](https://github.com/user-attachments/assets/f9434c7a-dedb-4e8f-a225-71fb65b4e586)
